File: laramie-tesoreria-webapi/src/domain/entities/registro-contable.ts
import Recaudacion from "./recaudacion";

export default class RegistroContable {

    id: number;
	idRegistroContableLote: number;
	idRecaudacion: number;
	fechaIngreso: Date;
	cuentaContable: string;
	jurisdiccion: string;
	recursoPorRubro: string;
	codigoLugarPago: string;
	ejercicio: string;
	codigoFormaPago: string;
	codigoTipoRecuadacion: string;
	importe: number;

	recaudacion: Recaudacion = null;

	constructor(
        id: number = 0,
		idRegistroContableLote: number = 0,
		idRecaudacion: number = 0,
		fechaIngreso: Date = null,
		cuentaContable: string = "",
		jurisdiccion: string = "",
		recursoPorRubro: string = "",
		codigoLugarPago: string = "",
		ejercicio: string = "",
		codigoFormaPago: string = "",
		codigoTipoRecuadacion: string = "",
		importe: number = 0
	)
	{
        this.id = id;
		this.idRegistroContableLote = idRegistroContableLote;
		this.idRecaudacion = idRecaudacion;
		this.fechaIngreso = fechaIngreso;
		this.cuentaContable = cuentaContable;
		this.jurisdiccion = jurisdiccion;
		this.recursoPorRubro = recursoPorRubro;
		this.codigoLugarPago = codigoLugarPago;
		this.ejercicio = ejercicio;
		this.codigoFormaPago = codigoFormaPago;
		this.codigoTipoRecuadacion = codigoTipoRecuadacion;
		this.importe = importe;
	}

	setFromObject = (row) =>
	{
        this.id = row.id ?? 0;
		this.idRegistroContableLote = row.idRegistroContableLote ?? 0;
		this.idRecaudacion = row.idRecaudacion ?? 0;
		this.fechaIngreso = row.fechaIngreso ?? null;
		this.cuentaContable = row.cuentaContable ?? "";
		this.jurisdiccion = row.jurisdiccion ?? "";
		this.recursoPorRubro = row.recursoPorRubro ?? "";
		this.codigoLugarPago = row.codigoLugarPago ?? "";
		this.ejercicio = row.ejercicio ?? "";
		this.codigoFormaPago = row.codigoFormaPago ?? "";
		this.codigoTipoRecuadacion = row.codigoTipoRecuadacion ?? "";
		this.importe = row.importe ?? 0;
	}

}

File: modules/tesoreria/models/registro_contable.py
from sqlalchemy import Column, BigInteger, String, Integer, Date, Numeric, ForeignKey
from sqlalchemy.orm import relationship

import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", ".."))
from shared.database import Base


class RegistroContable(Base):
    __tablename__ = "tesoreria_registros_contables"

    id = Column(BigInteger, primary_key=True)
    id_registro_contable_lote = Column(BigInteger, ForeignKey("tesoreria_registro_contable_lotes.id"), nullable=False)
    id_recaudacion = Column(BigInteger, nullable=True)
    fecha_ingreso = Column(Date, nullable=True)
    cuenta_contable = Column(String(250), nullable=True)
    jurisdiccion = Column(String(250), nullable=True)
    recurso_por_rubro = Column(String(250), nullable=True)
    codigo_lugar_pago = Column(String(50), nullable=True)
    ejercicio = Column(Integer, nullable=True)
    codigo_forma_pago = Column(String(50), nullable=True)
    importe = Column(Numeric(18, 2), nullable=False, default=0)

    lote = relationship("RegistroContableLote", back_populates="registros", lazy="joined")
